Resources/Prototypes/_Mono/Entities/ShipShield/shield.yml

```
@@ -0,0 +1,109 @@
+          True: { visible: true }
+          False: { visible: false }
+
+- type: entity
+  parent: BaseWall
+  id: GridShieldField
+  name: ship shield field
+  description: A shield that protects a ship.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: InteractionOutline
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)